src/test/java/lavalink/client/mock/MockLavalinkTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Written by TheMulti0, 2019
+ * https://github.com/TheMulti0
+ */
+
+package lavalink.client.mock;
+
+import com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager;
+import com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager;
+import com.sedmelluq.discord.lavaplayer.track.AudioItem;
+import com.sedmelluq.discord.lavaplayer.track.AudioPlaylist;
+import com.sedmelluq.discord.lavaplayer.track.AudioReference;
+import com.sedmelluq.discord.lavaplayer.track.AudioTrack;
+import lavalink.client.io.Lavalink;
+import lavalink.client.io.Link;
+import lavalink.client.io.mock.MockLavalink;
+import lavalink.client.player.LavalinkPlayer;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Timer;
+import java.util.TimerTask;
+
+public class MockLavalinkTest {
+
+    private static final Logger log = LoggerFactory.getLogger(MockLavalinkTest.class);
+    private final YoutubeAudioSourceManager youtube = new YoutubeAudioSourceManager();
+    private final DefaultAudioPlayerManager playerManager = new DefaultAudioPlayerManager();
+
+    @Test
+    public void play() throws InterruptedException {
+        Lavalink lavalink = new MockLavalink();
+        Link link = lavalink.getLink("435874356896543567");
+        LavalinkPlayer player = link.getPlayer();
+        player.playTrack(loadAudioTracks("Hi").get(0));
+
+        Timer timer = new Timer();
+        timer.schedule(new TimerTask() {
+            @Override
+            public void run() {
+                AudioTrack playingTrack = player.getPlayingTrack();
+                if (playingTrack == null) {
+                    timer.cancel();
+                } else {
+                    String format = String.format("Track %s is playing, %d / %d", playingTrack.getInfo().title, player.getTrackPosition(), playingTrack.getDuration());
+                    log.info(format);
+                }
+            }
+        }, 0, 100);
+        Thread.sleep(10000000);
+    }
+
+    private List<AudioTrack> loadAudioTracks(String identifier) {
```

Comment:
This is not an identifier but a search term. `ytsearch: foo` would be an identifier.

Reply:
I actually noticed that Lavaplayer uses the term `load` rather than `search` and `identifier` rather than `query` / `term` (because it does not specifically search), so wanted to work with the same names they do.

Reply:
That is because Lavalink is not just used for searching. An identifier can be a file path, a URL, or a search query. 

You could also save yourself the trouble and just mock the AudioTrack, rather than making the test rely on outside conditions.

Reply:
Hmm, great idea! Did not think of that.
And yeah I also said, identifier is not only a search query but can serve as a url, etc.